content/Dfe.EarlyYearsQualification.ContentUpload/Program.cs

```diff
@@ -15,6 +15,8 @@ public static class Program
     private const string SpaceId = "";
     private const string ManagementApiKey = "";
     private const string CsvFile = "./csv/ey-quals-full-2024-with-new-reference-fields.csv";
+    private const string QualificationContentTypeId = "Qualification";
```

Comment:
You could potentially use Dfe.EarlyYearsQualification.Content.Constants.ContentTypes.Qualification here. Not a blocking issue though

Reply:
Good suggestion, thank you. Done.